runtime/runtime/src/prefetch.rs

```diff
@@ -85,7 +89,10 @@ impl TriePrefetcher {
     /// Start prefetching data for processing the receipts.
     ///
     /// Returns an error if the prefetching queue is full.
```

Comment:
This comment has the danger of becoming obsolete and is also not complete.  I propose something like below .

`Returns an error if prefetching for any receipt fails.  The function is not idempotent; in case of failure, prefetching for some receipts may have been initiated.`

Reply:
lgtm, included